src/themes/teams/components/List/listItemStyles.ts

```diff
@@ -1,63 +1,15 @@
-import { pxToRem } from '../../../../lib'
-import { debugRoot } from '../../../../styles/debugStyles'
-
 const listItemStyles = {
-  root: ({ props, variables }) => {
-    const { debugLayout, important, selection } = props
-    return {
-      ...(debugLayout && debugRoot()),
-      gridTemplateRows: `minmax(${variables.height}, max-content)`,
-      paddingLeft: variables.paddingLeft,
-      paddingRight: variables.paddingRight,
-
-      ...(important && {
-        fontWeight: 'bold',
-      }),
-
-      ...(selection && {
-        position: 'relative',
-
-        ':hover': {
-          background: 'rgba(98, 100, 167, .8)',
-          color: '#fff',
-          cursor: 'pointer',
-        },
-      }),
-    }
-  },
-
-  media: ({ props }) => {
-    const { important } = props
-    return {
-      ...(important && {
-        '::before': {
-          content: '""',
-          position: 'absolute',
-          left: pxToRem(8),
-          width: pxToRem(2),
-          height: pxToRem(2),
-          background: '#000',
-        },
-      }),
-    }
-  },
-
-  header: ({ props, variables }) => ({
-    lineHeight: variables.headerLineHeight,
-  }),
-
-  headerMedia: ({ props, variables }) => ({
-    color: variables.headerMediaColor,
-    fontSize: variables.headerMediaFontSize,
-    lineHeight: variables.headerMediaLineHeight,
-  }),
-
-  content: ({ props, variables }) => ({
-    color: variables.contentColor,
-    fontSize: variables.contentFontSize,
-    lineHeight: variables.contentLineHeight,
+  root: ({ props: { selection } }) => ({
+    ...(selection && {
+      position: 'relative',
+
+      ':hover': {
+        background: 'rgba(98, 100, 167, .8)',
+        color: '#fff',
+        cursor: 'pointer',
+      },
+    }),
```

Comment:
We should move the non-position related styles back to the ListItem from the Layout.  The layout should not define colors, fonts, etc but only layout styling.